WebRoot/messages/ZtMsg_sk.properties

```diff
@@ -0,0 +1,16 @@
+#
```

Comment:
it's ok to not add these files for ZtMsg, it's for sencha touch and it's deprecated/removed from zimbra 9

Reply:
sure